tests/unit/integrations/front-end/crawl-cleanup-searches-test.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Integrations\Front_End;
+
+use Brain\Monkey;
+use Mockery;
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Integrations\Front_End\Crawl_Cleanup_Searches;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+use Yoast\WP\SEO\Helpers\Redirect_Helper;
+use WP_Query;
+
+/**
+ * Class Crawl_Cleanup_Searches_Test.
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Integrations\Front_End\Crawl_Cleanup_Searches
```

Comment:
Coverage could be improved.
P.S. you can add `@covers` private methods when the public method that is called calls it

Reply:
> P.S. you can add @covers private methods when the public method that is called calls it
Right, didn't think about that! 👍